mlrun/model.py

```diff
@@ -508,12 +630,22 @@ def __init__(
         self.scrape_metrics = scrape_metrics
         self.allow_empty_resources = allow_empty_resources
 
-    def to_dict(self, fields=None, exclude=None):
-        struct = super().to_dict(fields, exclude=["handler"])
+    def to_dict(self, fields: list = None, exclude: list = None, strip: bool = False):
```

Comment:
looks like you don't need to override the method because you already handle this in the `_serialize_field` method